docs/content/configuration/filter.md

```diff
@@ -68,3 +68,19 @@ D enterprise.github.com/company
 </pre>
 
 In the above example, any module not in the rules will be excluded.  All modules from `enterprise.github.com/company` are fetched directly from the source.  The `github.com/gomods/athens` module will be stored in the proxy storage, but only for version `v0.4.1` and any patch versions under `v0.1` and `v0.2` minor versions.
+
+### Versions Filter Modifiers
+
+Athens provides advanced filter modifiers to cover cases such as api compatibility or when a given dependency changes its license from a given versions. The modifiers are intended to be used in the pattern list of the filter file.
+
+<pre>
+-
+# external dependency approved list
++ github.com/gomods/athens <1.2.3
+</pre>
+
+The currently supported modifiers are 
+
+- ~1.2.3: enables 1.2.x  which are at least 1.2.3
```

Comment:
Out of curiosity: why aren't we using the `v` prefix for semver tags just like go users expect them to be?

Reply:
That's reasonable (and now I remember you said the same thing on the issue). Will change that to ~v1.2.3 and such.

Reply:
Changed